scripts/catalog-to-astro-content-directory.js

```diff
@@ -9,7 +9,7 @@ const scriptsDir = path.dirname(__filename);
 
 const getTargetPath = (source, target, type, file) => {
   const relativePath = path.relative(source, file);
-  const cleanedRelativePath = relativePath.split(type);
+  const cleanedRelativePath = relativePath.split(`${type}/`);
```

Comment:
The `/` was removed for compatibility with `win32` plataform.
Maybe `path.sep` could works 🤷🏼‍♂️. 

When I worked at a refactor proposal for the `catalog-to-astro-content-directory.js` I got the following for this snippet:
```ts
/**
 * Check if the key is an ASTRO COLLECTION KEY
 * @param {string} key
 * @returns {boolean}
 */
const isCollectionKey = (key) => {
  const COLLECTION_KEYS = ['commands', 'domains', 'events', 'pages', 'services', 'teams', 'users'];
  /** 
   *  Using includes won't works for your use case either
   *  The solution will be a comparison
   */
  return COLLECTION_KEYS.includes(key);
};

/**
 * Generates the path until the ASTRO_COLLECTION_KEY or the PROJECT_DIR root.
 * @param {string} filePath The path to the file.
 * @returns {string} The path until the COLLECTION_KEY or PROJECT_DIR root.
 */
const getRelativeTargetPath = (filePath) => {
  const filePathParsed = path.parse(filePath);
  const fileDir = filePathParsed.dir.split(path.sep).filter(Boolean);

  const relativePath = [];
  for (let i = fileDir.length - 1; i >= 0; i--) {
    relativePath.unshift(fileDir[i]);
    if (isCollectionKey(fileDir[i])) break;
  }

  return path.join(...relativePath, filePathParsed.base);
};
```
You can check the implementation [here](https://gist.github.com/carlosallexandre/9a410d493b4f4f63d54157642395381d). 
I hope this can help you find a suitable solution.

Reply:
Thanks @carlosallexandre for review. I committed new changes, can you check?